pkg/graph/resolvers/system_intake_actions.go

```diff
@@ -529,13 +530,28 @@ func CreateSystemIntakeActionReopenRequest(
 			return nil, err
 		}
 	}
+	if emailClient != nil && input.NotificationRecipients != nil { // Don't email if no recipients are provided or there isn't an email client
+		err = emailClient.SystemIntake.SendReopenRequestNotification(ctx,
+			*input.NotificationRecipients,
+			intake.ID,
+			intake.ProjectName.ValueOrZero(),
+			intake.Requester,
+			*input.Reason,
```

Comment:
This might cause a nil-dereference error; `Reason` is an optional field in the UI and in the GraphQL schema (hence why `input.Reason` is a `*models.HTML` pointer). Reason should presumably be an optional field in the email as well, so you'll need to make that field a pointer as it's passed to the email template and treat it like `AdditionalInfo`.

Reply:
Good catch, thank you.